zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiAutoConfiguration.kt

```diff
@@ -86,13 +88,24 @@ class ObjectenApiAutoConfiguration {
     }
 
     @Bean
+    @ConditionalOnMissingBean(ZaakObjectDataResolver::class)
     fun zaakObjectDataResolver(
```

Comment:
Why is this a bean anyway? shouldn't the factory make one?

Reply:
Because this kind of resolver is for forms and doesn't need a factory.
`zaakObjectDataResolver != zaakObjectValueResolver`

Reply:
Fair enough